radix-engine/src/engine/process.rs

```diff
@@ -946,6 +971,42 @@ impl<'r, 'l, L: SubstateStore> Process<'r, 'l, L> {
         Ok(CallMethodOutput { rtn: result?.raw })
     }
 
+    fn move_lazy_map_into_lazy_map(
+        &mut self,
+        unclaimed_lazy_map: UnclaimedLazyMap,
+        mid: Mid,
+        mid_parent: &Mid,
+    ) {
+        let parent_lazy_map = self.unclaimed_lazy_maps.get_mut(&mid_parent).unwrap();
+        parent_lazy_map
+            .child_lazy_maps
+            .insert(mid, unclaimed_lazy_map.lazy_map);
+
+        for (mid, lazy_map) in unclaimed_lazy_map.child_lazy_maps {
+            parent_lazy_map.child_lazy_maps.insert(mid, lazy_map);
+        }
+        for (vid, vault) in unclaimed_lazy_map.child_vaults {
+            parent_lazy_map.child_vaults.insert(vid, vault);
+        }
+    }
+
+    fn move_lazy_map_into_component(
+        &mut self,
+        unclaimed_lazy_map: UnclaimedLazyMap,
+        mid: Mid,
+        component_address: Address,
+    ) {
+        self.track
```

Comment:
I'd suggest we add some logs to the two move methods. This is extremely useful for debugging through logs (without fancy IDEs).